### PR TITLE
feat(ps): Add Ballots Scanned count to more pages

### LINKED ISSUE #2282: VxScan: add ballot count to all screens

### PR DESCRIPTION
**Pull request by beausmith**
## Overview
Closes #2282 

I didn't add the count to "modal" pages where the voter needs to make choice, or screens instructing the user to remove a card.

### CODE REVIEW

File: frontends/precinct-scanner/src/screens/polls_closed_screen.tsx [hunk 1]
```diff
@@ -28,26 +31,51 @@ export function PollsClosedScreen({
...
 }
 
 /* istanbul ignore next */
 export function DefaultPreview(): JSX.Element {
-  return <PollsClosedScreen isLiveMode showNoChargerWarning={false} />;
+  return (
+    <PollsClosedScreen
+      isLiveMode
+      showNoChargerWarning={false}
+      scannedBallotCount={42}
```
Comment (arsalansufi) [inline frontends/precinct-scanner/src/screens/polls_closed_screen.tsx:45]: Nice demo number choice 😄

File: frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx [hunk 2]
```diff
@@ -28,16 +33,19 @@ export function SetupScannerScreen({ batteryIsCharging }: Props): JSX.Element {
           </Text>
         </CenteredLargeProse>
       )}
+      {scannedBallotCount && <ScannedBallotCount count={scannedBallotCount} />}
```
Comment (arsalansufi) [inline frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx:36]: What if the `scannedBalotCount` is `0`? We currently won't display a count in that case - is that what we want?

**Review by arsalansufi — COMMENTED** [review]
A quick sanity check!

File: frontends/precinct-scanner/src/screens/polls_closed_screen.tsx (hunk 1 above)
Comment (beausmith) [reply frontends/precinct-scanner/src/screens/polls_closed_screen.tsx:45]: Making sure we know its fake!

File: frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx (hunk 2 above)
Comment (beausmith) [reply frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx:36]: 🙏  Fixing…

**Review by arsalansufi — APPROVED** [review]
LGTM!

**Pull request merged**